packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -1,27 +1,33 @@
 import '../../solid-components';
 import { html } from 'lit';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-dropdown');
 const { generateTemplate } = storybookTemplate('sd-dropdown');
 const { overrideArgs } = storybookHelpers('sd-dropdown');
 
+/**
+ * Used to display a list of actions or options in a panel when activated.
+ *
+ * **Related templates:**
```

Comment:
For the 1st sample can we used the "hovered" version as we do in FIGMA?

Reply:
Already discussed with @smfonseca ... sd-navigation-item needs to be updated in order to be able to display that. Do we already have an issue for that?

Reply:
We do => [#1352](https://github.com/solid-design-system/solid/issues/1352)